add how to clean up the e2e connectivity test. (#30428)
cilium/cilium: Documentation/contributing/testing/e2e.rst

    # make modules_install

Finally, you can use the instructions from :ref:`the previous chapter<test_cilium_on_lvh>` to run and to test Cilium.

Cleaning up tests
^^^^^^^^^^^^^^^^^

> For cleaning up, you can simply delete the namespace for connectivity tests.

Reviewer: ```suggestion
To clean up, simply delete the connectivity tests namespace:
```
A suggestion for simpler language.

> For cleaning up, you can simply delete the namespace for connectivity tests.

Reviewer: My understanding is that if the tests complete then the namespace will be deleted automatically, so this cleanup is necessary only if you interrupt tests? If that's correct, maybe it's worth mentioning it here.

.. code-block:: shell-session

    $ kubectl delete ns cilium-test

> if test namespace is specified with `--test-namespace`, make sure replace ``cilium-test`` (default).

Reviewer: ```suggestion
If you specified the test namespace with ``--test-namespace``, make sure to replace ``cilium-test`` (default).
```
A style suggestion.
Also, the CI is failing (https://github.com/cilium/cilium/actions/runs/7646311701/job/20834853707?pr=30428) because of single backticks here, they should be double.
